File: examples/python/list_events.py
import requests

res = requests.get(
    "https://api.sealtrail.dev/v1/events",
    headers={"Authorization": "Bearer stl_live_..."},
    params={"actor": "user_123", "after": "2026-03-01T00:00:00Z", "limit": 50},
)

body = res.json()
events = body["data"]
next_cursor = body.get("nextCursor")  # None if last page

File: examples/typescript/list-events.ts
const params = new URLSearchParams({
	actor: "user_123",
	after: "2026-03-01T00:00:00Z",
	limit: "50",
});

const res = await fetch(`https://api.sealtrail.dev/v1/events?${params}`, {
	headers: { Authorization: "Bearer stl_live_..." },
});

const { data, nextCursor } = await res.json();
// data: Event[]
// nextCursor: "eyJjcmVhdGVkQXQiOi4uLn0=" (if more pages)
